info/v1/container.go

```diff
@@ -306,6 +306,11 @@ type MemoryStats struct {
 	// Units: Bytes.
 	Usage uint64 `json:"usage"`
 
+	// Various statistics reported in memory.stat in cgroupfs. See the section
+	// "5.2 stat file" of https://www.kernel.org/doc/Documentation/cgroups/memory.txt
+	// for details.
+	Stats map[string]uint64 `json:"stats"`
```

Comment:
The v2 API references v1: https://github.com/google/cadvisor/blob/e4262b91b17cc132a35a023865f9ef97a048aa21/info/v2/container.go#L102